### PR TITLE
update postprocess.sh / Update run.py

### PR DESCRIPTION
**Pull request by rumenmitrev**
update postprocess.sh and resolve issue/35

### CODE REVIEW

**Comment by dronemapper-io** [comment]
Thanks for the PR. :) I will review and test it asap.. then update the docker hub image.

**Comment by rumenmitrev** [comment]
Be sure to test it with latest MicMac.

File: dm/run.py [hunk 1]
```diff
@@ -363,15 +363,15 @@ def remove_ortho_tiles():
...
-        #filter TiePoints (better distribution, avoid clogging)
-        #kwargs_schnaps = {
-        #    'ext': image_ext,
-        #    'mm3d': mm3d
-        #    
-        #}
-	#    system.run('{mm3d} Schnaps .*.{ext} MoveBadImgs=1'.format(**kwargs_schnaps))
-        #
-        #progressbc.send_update(5)
+        filter TiePoints (better distribution, avoid clogging)
```
Comment (dronemapper-io) [inline dm/run.py]: we can try this but in my experience schnaps has some issues. also, we want to stick with RadialStd as it is more flexible for calibration.. i'll run some tests and we like were you are going with these enhancements

**Comment by dronemapper-io** [comment]
Sorry, been crazy busy but will try and get this wrapped up this weekend

**Comment by rumenmitrev** [comment]
I`m not in hurry at all.
:)

File: dm/run.py (hunk 1 above)
Comment (rumenmitrev) [reply dm/run.py]: The problem with Schnaps is that , need to load whole Homol folder in RAM

**Pull request closed without merging**